artemis-web/src/main/java/org/apache/activemq/artemis/ActiveMQWebLogger.java

```diff
@@ -57,4 +57,8 @@
    @LogMessage(level = Logger.Level.WARN)
    @Message(id = 244003, value = "Temporary file not deleted on shutdown: {0}", format = Message.Format.MESSAGE_FORMAT)
    void tmpFileNotDeleted(File tmpdir);
+
+   @LogMessage(level = Logger.Level.DEBUG)
```

Comment:
We don't need to use the Logger for Debug Messages...


just make these calls



on the declarion of the class: private static final Logger logger = Logger.getLogger(...); // use the right package here.. and there are other examples you can follow...



and then log.debug(".... .static text...")



These messages are for INFO, WARN, ERROR.. or things that will need a code to be printed out.